src/GitVersionTask/GitVersionTask.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>GitVersionTask</RootNamespace>
     <AssemblyName>GitVersionTask</AssemblyName>
-    <TargetFrameworks>net461;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Can `net461` now be removed here, as net461 implements netstandard20?

Reply:
Yeah I wasn't really sure if it was OK to remove it in the task project or not because I didn't have any real experience with it so I decided it was safer to leave it alone until I was told otherwise. :)

Reply:
As far as I know, that way it can be referenced from other .NET Standard libs and doesn't cause problems with an implicit reference to the full .NET Framework

Reply:
@li0nsar3c00l That isn't how referencing multi-targeted nuget packages works. The project system will do the right thing and only pull in the most applicable version of the referenced library. If you reference it from a netstandard2 project you will only end up with a reference to the netstandard2 version of the library.